test.sh.stdout

```diff
@@ -0,0 +1,190 @@
+dummy.img23: Created a new partition 23 of type 'unknown' and of size 4 KiB.
+dummy.img24: Created a new partition 24 of type 'unknown' and of size 4 KiB.
+dummy.img25: Created a new partition 25 of type 'unknown' and of size 4 KiB.
+dummy.img26: Created a new partition 26 of type 'unknown' and of size 4 KiB.
+dummy.img27: Created a new partition 27 of type 'unknown' and of size 4 KiB.
+dummy.img28: Created a new partition 28 of type 'unknown' and of size 4 KiB.
+dummy.img29: Created a new partition 29 of type 'unknown' and of size 4 KiB.
+dummy.img30: Created a new partition 30 of type 'unknown' and of size 4 KiB.
+dummy.img31: Created a new partition 31 of type 'unknown' and of size 4 KiB.
+dummy.img32: Created a new partition 32 of type 'unknown' and of size 4 KiB.
+dummy.img33: Created a new partition 33 of type 'Linux filesystem' and of size 19.8 GiB.
+All partitions used.
+
+New situation:
+Disklabel type: gpt
+Disk identifier: F4796A2A-E377-45BD-B539-D6D49E569055
+
+Device       Start      End  Sectors  Size Type
+dummy.img1      32   409623   409592  200M EFI System
+dummy.img2  411648   415743     4096    2M BIOS boot
+dummy.img3  415744   419839     4096    2M Linux filesystem
+dummy.img4  419840   423935     4096    2M Linux filesystem
+dummy.img5  423944   423951        8    4K unknown
+dummy.img6  423952   423959        8    4K unknown
+dummy.img7  423960   423967        8    4K unknown
+dummy.img8  423968   423975        8    4K unknown
+dummy.img9  423976   423983        8    4K unknown
+dummy.img10 423984   423991        8    4K unknown
+dummy.img11 423992   423999        8    4K unknown
+dummy.img12 424000   424007        8    4K unknown
+dummy.img13 424008   424015        8    4K unknown
+dummy.img14 424016   424023        8    4K unknown
+dummy.img15 424024   424031        8    4K unknown
+dummy.img16 424032   424039        8    4K unknown
+dummy.img17 424040   424047        8    4K unknown
+dummy.img18 424048   424055        8    4K unknown
+dummy.img19 424056   424063        8    4K unknown
+dummy.img20 424064   424071        8    4K unknown
+dummy.img21 424072   424079        8    4K unknown
+dummy.img22 424080   424087        8    4K unknown
+dummy.img23 424088   424095        8    4K unknown
+dummy.img24 424096   424103        8    4K unknown
+dummy.img25 424104   424111        8    4K unknown
+dummy.img26 424112   424119        8    4K unknown
+dummy.img27 424120   424127        8    4K unknown
+dummy.img28 424128   424135        8    4K unknown
+dummy.img29 424136   424143        8    4K unknown
+dummy.img30 424144   424151        8    4K unknown
+dummy.img31 424152   424159        8    4K unknown
+dummy.img32 424160   424167        8    4K unknown
+dummy.img33 424168 41943015 41518848 19.8G Linux filesystem
+
+The partition table has been altered.
+label: dos
+label-id: 0x00000000
+device: /dev/loop0
+unit: sectors
+sector-size: 4096
+
+/dev/loop0p1 : start=           1, size=     5242879, type=ee
```

Comment:
This is I assume expected output when reading with mismatching sector size. Might be better to force sfdisk to look for gpt (`--label gpt`) and expect it to fail before calling the gpt fix tool.